### PR TITLE
added directory for random scripts, not to be copied to CCP4 source i…

### PR DESCRIPTION
**Pull request by fsimkovic**
…nstallation

### CODE REVIEW

File: scripts/stats.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+from __future__ import print_function
+
+__author__ = "Felix Simkovic"
+__date__ = "30 Mar 2018"
+__version__ = "1.0"
+
+import os
+import re
+import requests
```
Comment (hlasimpk) [inline scripts/stats.py]: In my ccp4-python, requests can't be imported which might cause errors

Comment (fsimkovic) [reply scripts/stats.py]: Not sure this is so important right now. This script is really just for us and could be a data dump for database sizes etc ...

Comment (hlasimpk) [reply scripts/stats.py]: See my message on slack

Comment (hlasimpk) [reply scripts/stats.py]: The urllib2 code would look like this:

req = urllib2.Request(url, data, header)
response = urllib2.urlopen(req)

if response.getcode() == 200:
	count = len(response.readlines())

**Pull request merged**